libensemble/resources/resources.py

```diff
@@ -356,9 +356,10 @@ def get_local_nodelist(num_workers, workerID, resources):
 
         if workerID in zero_resource_list:
             local_nodelist = []
+            logger.debug("Worker {} is a zero-resource worker")
         else:
             index = WorkerResources.map_workerid_to_index(num_workers, workerID, zero_resource_list)
             local_nodelist = split_list[index]
+            logger.debug("Worker {}'s local_nodelist is {}".format(workerID, local_nodelist))
```

Comment:
I'm fine with this if it makes it clearer, but note that worker ID should appear at the start of the line.

Reply:
Ah, you're right. I accidentally overlooked those IDs. Thanks